experimental/learn_descriptors/four_seasons_parser_example.cc

```diff
@@ -0,0 +1,70 @@
+#include <cstddef>
+#include <filesystem>
+#include <iostream>
+#include <opencv2/opencv.hpp>
+#include <sstream>
+#include <string>
+#include <vector>
+
+#include "experimental/learn_descriptors/four_seasons_parser.hh"
+
+namespace lrn_desc = robot::experimental::learn_descriptors;
+
+int main(int argc, char* argv[]) {
+    std::vector<std::string> args(argv, argv + argc);
+    std::filesystem::path path_data, path_calibration;
+
+    for (int i = 0; i < argc; i++) {
+        std::string arg = argv[i];
+        std::cout << arg << std::endl;
+        if (arg == "--data_dir") {
+            std::cout << "DATA DIR PROVIDED!" << std::endl;
+            path_data = std::filesystem::path(std::string(argv[++i]));
+        } else if (arg == "--calibration_dir") {
+            std::cout << "CONFIG DIR PROVIDED!" << std::endl;
+            path_calibration = std::filesystem::path(std::string(argv[++i]));
+        }
+    }
```

Comment:
I've used `cxxopts.hpp` to parse cli args. See `experimental/pokerbots/pokerbot_master.cc` for an example.